### ISSUE TITLE
Tiny Yolo- V2 -voc training

### ISSUE THREAD

**Issue by tsubu01** [pjreddie/darknet#1451]
Hey!
I cloned the repo, and trained the net described in yolov2-tiny-voc.cfg as-is, as well as in variations on the basic learning rate, but could not achieve a net with MAP=57% (or even 54.5%, as given by the supplied checkpoint).
I am using the training and testing sets as given (training set is the union of 2007 train, 2007 val and 2012 train, total size ~ 16k images, test set is 2007 test, total size ~ 5k images). 
In fact, I reach values closer to 40%...
What am I doing wrong?
Thanks!
